yw_website/yw_website/apps/website/tests.py

```diff
@@ -319,3 +320,146 @@ def test_no_repeating_tags(self):
+        self.data["data"] = [
+            {
+                "dataId": 1,
+                "inProgramBlock": 1,
+                "name": "data1",
+                "qualifiedName": "dataQualifiedName",
+            },
+            {
+                "dataId": 2,
+                "inProgramBlock": 1,
+                "name": "dataName2",
+                "qualifiedName": "dataQualifiedName2",
+            },
+        ]
+        self.data["port"] = [
+            {
+                "portId": 1,
+                "onProgramBlock": 1,
+                "data": 1,
+                "name": "PortOne",
+                "qualifiedName": "portOneQualifiedName",
+                "alias": None,
+                "uriTemplate": None,
+                "inPort": True,
+                "outPort": False,
+            },
+            {
+                "portId": 2,
+                "onProgramBlock": 2,
+                "data": 2,
+                "name": "PortTwo",
+                "qualifiedName": "portTwoQualifiedName",
+                "alias": "aliasTwo",
+                "uriTemplate": "uriTwo",
+                "inPort": False,
+                "outPort": True,
+            },
+        ]
+        self.data["channel"] = [
+            {
+                "channelId": 1,
+                "inPort": 1,
+                "outPort": 2,
+                "data": 1,
+                "isInflow": True,
+                "isOutflow": False,
+            }
+        ]
+        self.data["uriVariable"] = [
+            {"uriVariableId": 1, "port": 1, "name": "urivarname"}
+        ]
+        self.data["resource"] = [
+            {
+                "resourceId": 1,
+                "data": 1,
+                "uri": "file_uri_1",
+                "name": "file_name_1",
+                "checksum": str(uuid.uuid1()),
+                "size": 9,
+                "lastModified": datetime.datetime.now(),
+            },
+            {
+                "resourceId": 2,
+                "data": 1,
+                "name": "file_name_2",
+                "checksum": str(uuid.uuid1()),
+                "size": 3,
+                "uri": "file_uri_2",
+                "lastModified": datetime.datetime.now(),
+            }
+        ]
+        self.data["uriVariableValue"] = [
+            {"uriVariableId": 1, "resource": 1, "value": "uripath"}
+        ]
+
+        route = "/save/"
+        response = self.client.post(route, self.data, format="json")
+
+    def test_view_home(self):
+        response = self.client.get(reverse('home'))
+        self.assertEqual(response.status_code, 200, msg="The home page is not returning a 200 status code")
+
+    def test_view_my_workflows_authenticated(self):
+        response = self.client.get(reverse('my_workflows'))
+        self.assertEqual(response.status_code, 200,
+                         msg="An unathenticated user was able to access my_workflows page")
+
+    def test_view_my_workflows_unauthenticated(self):
+        self.client.logout()
```

Comment:
this might effect other tests that require authenticated client, you should create a new client object instead of logging out this one

Reply:
> you should create a new client object

and make sure that the new client object is logged out